### ISSUE TITLE
Wrong match

### ISSUE THREAD

**Issue by stoecker** [Hubird-au/Adversity#1]
Hello, when going to http://d-nb.info/1068359102 this list filters out the display of the image. Could you please correct this, as this is a false positive.

Also the homepage link http://adversity.uk.to/ is broken: "Host adversity.uk.to not found: 3(NXDOMAIN)".

**Comment by stoecker** [comment]
Actually after looking at the code maybe the match is not totally wrong. Still your website is broken.

**Comment by Hubird-au** [comment]
Yeah, website is down and I maintain the list only for my personal use now
